### PR TITLE
New version: Seewo.EasiNote version 5.2.3.5046

### PR DESCRIPTION
**Pull request by SpecterShell**
This pull request is automatically generated by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings)
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131361)

### CODE REVIEW

File: manifests/s/Seewo/EasiNote/5.2.3.5046/Seewo.EasiNote.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Protocols:
+- easinote
+FileExtensions:
+- enb
+- enbx
+- enpx
+ReleaseDate: 2023-12-20
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cstore-pub-cos-seewo-report-tx.seewo.com/seewo-report_uwixvoojhhulnhlwhlmhuxnowyhhihhh?sign=q-sign-algorithm%3Dsha1%26q-ak%3DAKIDJUXMxJRLzPaeMp20jDSTFl23pLcdPwDF%26q-sign-time%3D1703040686%3B2018400686%26q-key-time%3D1703040686%3B2018400686%26q-header-list%3Dhost%26q-url-param-list%3Dresponse-content-disposition%26q-signature%3D66e79c05b4bc1bf62e6231b874b33dc44294a4d5&response-content-disposition=attachment%3Bfilename%3D%22EasiNoteSetup_5.2.3.5046_seewo.exe%22%3Bfilename%2A%3Dutf-8%27%27EasiNoteSetup_5.2.3.5046_seewo.exe
```
Comment (stephengillie) [inline manifests/s/Seewo/EasiNote/5.2.3.5046/Seewo.EasiNote.installer.yaml:18]: This InstallerUrl seems to have a time-based component.

**Comment by SpecterShell** [comment]
(Hold on for some days)

File: manifests/s/Seewo/EasiNote/5.2.3.5046/Seewo.EasiNote.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/s/Seewo/EasiNote/5.2.3.5046/Seewo.EasiNote.installer.yaml:18]: Maybe not. This URL comes from their API without any redirections. You can still access to the early versions of installers.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**